makefile

```diff
@@ -23,14 +23,27 @@
 #                    - Fixed default rule so it works on NetBSD, Tru64, and
 #                      Linux (now we can get rid of make.sh) - MT
 #  01 Apr 24         - Consolidate Linux/NetBSD/Darwin and OSF1 - macmpi
+#  02 Apr 24         - Add differentiated desktop files installs - macmpi
 #
 
 all:
-	@$(MAKE) -s -f "makefile.common" $@ ;
-
-backup:
-	@$(MAKE) -s -f "makefile.backup" $@ ;
+	@$(MAKE) -s -f "makefile.common" $@
 
 .DEFAULT:
-	@$(MAKE) -s -f "makefile.common" $@ ;
+	@$(MAKE) -s -f "makefile.common" $@
+
+clean:
+	@$(MAKE) -s -f "makefile.common" $@
+
+install:
+	@$(MAKE) -s -f "makefile.common" $@
+	@_flavor="`uname | tr '[:upper:]' '[:lower:]'`"; \
+	if [ "$$_flavor" = "osf1" ]; then \
```

Comment:
Might to be better to test based on the existence of $HOME/.local ?  That way if a window manager is installed that is NOT freedesktop compliant (like TWM) the installer won't create the files there?

Reply:
Nice idea - could we use `DESKTOP=cde` `DESKTOP=freedesktop` and `DESKTOP=none` to override the local setup?  Think this would be easier to extend later if someone wanted to..

Reply:
so we'd have `DESKTOP=default` (use local target plaform's default),  `DESKTOP=none`, or force `DESKTOP=cde` `DESKTOP=freedesktop`  (and other eventual custom) whatever platform is built upon, right?

Reply:
Yes but I'd also use the local target platform's default if DESKTOP` was not defined (ie the default).  

That way when packaging for Flathub it should just work as it is...

Reply:
To avoid getting out of hand I think that if the `$HOME/.local` exists we should assume it is a freedesktop environment otherwise `NONE` if `DESKTOP` is undefined ?